src/molecules/BottomSheet/BottomSheet.native.js

```diff
@@ -0,0 +1,194 @@
+import React, { forwardRef, useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { Dimensions } from 'react-native';
+import styled, { useTheme } from 'styled-components/native';
+import { Modalize as RNModalize } from 'react-native-modalize';
+import LinearGradient from 'react-native-linear-gradient';
+import View from '../../atoms/View';
+import Flex from '../../atoms/Flex';
+import Space from '../../atoms/Space';
+import Size from '../../atoms/Size';
+import Divider from '../../atoms/Divider';
+import reactChildrenGroupByType from '../../_helpers/reactChildrenGroupByType';
+import Position from '../../atoms/Position';
+import BottomSheetHeader from './BottomSheetHeader';
+import BottomSheetFooter from './BottomSheetFooter';
+import BottomSheetContent from './BottomSheetContent';
+
+const screenHeight = Dimensions.get('window').height;
+const DEFAULT_SNAP_POINT = screenHeight * 0.4; // 40% of screen height
+const linearGradientLocations = [0, 0.2969, 1]; // 0%, 29.69%, 100%
+
+const styles = {
+  rootStyle: () => {
+    return {
+      elevation: 5, // to be removed once Home TabsBottomSheet is changed
+    };
+  },
+  overlayStyle: ({ theme }) => {
+    return {
+      backgroundColor: theme.colors.shade[950],
+    };
+  },
+  childrenStyle: ({ theme }) => {
+    return {
+      backgroundColor: theme.colors.background[200],
+    };
+  },
+  linearGradient: () => {
+    return {
+      transform: [
+        {
+          scaleX: 1,
+        },
+        { scaleY: -1 },
+      ],
+    };
+  },
+};
+
+const HeaderContainer = styled(View)`
+  background-color: ${(props) => props.theme.colors.background[200]};
+  border-top-right-radius: ${(props) => props.theme.spacings.small};
+  border-top-left-radius: ${(props) => props.theme.spacings.small};
+  box-shadow: ${(props) => `0px -4px 15px ${props.theme.colors.primary[930]}`};
+  elevation: 5;
```

Comment:
Is this the correct elevation here? I thought we were using it only above

Reply:
I kept this to have the shadow at the top. Once the rootStyle is removed

Reply:
Wait, what if we keep elevation in rootStyle and remove it from here? Anyways it is required for android.

Reply:
Yeah do that